### PR TITLE
Add daily management command to send notification email when applications open.

### PR DESCRIPTION
**Pull request by sarahboyce**
(need to set up the cron job and generally agree this is wanted)
image

### CODE REVIEW

File: home/management/commands/applications_open_email.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        except Session.DoesNotExist:
+            self.stdout.write(
+                self.style.SUCCESS(
+                    "There are no sessions with applications starting today"
+                )
+            )
+            return False
+        emails_sent = 0
+        for user in (
+            CustomUser.objects.select_related("profile")
```
Comment (tim-schilling) [inline home/management/commands/applications_open_email.py:32]: Do we need to select profile here? I don't see it being used in the formation of `email_data`.

**Review by tim-schilling — APPROVED** [review]
Looks good!

File: home/management/commands/applications_open_email.py (hunk 1 above)
Comment (sarahboyce) [reply home/management/commands/applications_open_email.py:32]: `user.profile.create_unsubscribe_link()`

Comment (tim-schilling) [reply home/management/commands/applications_open_email.py:32]: D'oh. Thank you!

**Pull request merged**